pages/super-collaborator/run.html

```diff
@@ -172,6 +173,18 @@
         hpccWasm.graphviz.layout(window.dot, "svg", "dot").then(svg => {
           target.innerHTML = svg;
           drawing = false
+          const targetBounds = { width: document.querySelector('#target').clientWidth, height: document.querySelector('#target').clientHeight } 
+          const svgBounds = { width: document.querySelector('#target svg').clientWidth, height: document.querySelector('#target svg').clientHeight }
+          // console.log(targetBounds, svgBounds)
+          let svgElement = document.querySelector('#target svg')
+          let panSVG = svgPanZoom(svgElement)
+          document.querySelector('#target svg').style.height = "100%"
+          document.querySelector('#target svg').style.width = "100%"
```

Comment:
Set the style here, after the SVG has been added, rather than in the stylesheet. This prevents the SVG from zooming automatically, for example a single node filling the entire target.

Reply:
Can we preserve the pan and zoom as we pass through this code? I loose the user-selected view when we set this to 100%. When I remove these lines we still remove the user-selected view by returning to the previous default. This was not noticeable when there was no user-selected view.

When looking for interactions between large graphs one prefers the gold/green color scheme as gold marks interactions. When one interaction has been brought into view one would like to switch to the application specific color scheme, for example Reimage Colors. But this small change looses the very positioning that brought the interaction of interest into view.

Reply:
Excellent behavior in all cases now.